src/main/java/de/bonndan/nivio/assessment/StatusValue.java

```diff
@@ -10,7 +10,7 @@
 
 /**
  * One specific property/kpi/key ... carrying a status.
- *
+ * <p>
```

Comment:
😄 ask @jenarp how to remove that

Reply:
These <p> originate from @BriceNtchoumou commits and is in my IntelliJ solved since 20.07.2021
![image](https://user-images.githubusercontent.com/84507051/143049595-720e62b4-b049-4deb-a2b7-562626144f4c.png)